### PR TITLE
use pandas to calculate the value results

### PR DESCRIPTION
**Pull request by jmelis**

### CODE REVIEW

File: utils/create-gnuplot-values-file.py [hunk 1]
```diff
@@ -1,53 +1,58 @@
...
+
+    conn_errors = (df['err'] != '')
+    total_conn_errors = len(conn_errors[conn_errors==True])
+
+    total_requests = len(df)
+    total_errors = total_http_errors + total_conn_errors
+
+    start_time = df['start_request'][total_requests - 1]
+    end_time = df['start_request'][0]
+    attack_duration = start_time - end_time
```
Comment (rporres) [inline utils/create-gnuplot-values-file.py]: `start_time` and `end_time` are reversed

**Review by rporres — COMMENTED** [review]
A small thing.

Apart from that, since we're using non-core libraries we should consider a `setup.py` file or something to declare dependencies at some point. It doesn't have to be now :)

File: utils/create-gnuplot-values-file.py (hunk 1 above)
Comment (jmelis) [reply utils/create-gnuplot-values-file.py]: good catch! changing.

About setup.py, is it ok if I just add a requirements.txt for the moment?

Comment (rporres) [reply utils/create-gnuplot-values-file.py]: Yes, `requirements.txt` is ok

**Review by rporres — APPROVED** [review]

**Pull request merged**